integration/build.gradle

```diff
@@ -16,6 +16,7 @@ dependencies {
     implementation 'org.slf4j:slf4j-api'
     implementation 'org.glassfish.jersey.inject:jersey-hk2'
     implementation 'org.junit.jupiter:junit-jupiter-api'
+    implementation 'org.hamcrest:hamcrest'
```

Comment:
Is there a strong reason to switch to hamcrest for testing? Easier use of assertions? I see without it the assertions are a few more lines of code, but the perhaps the tradeoff of adding a new package is worth it?

Reply:
In most cases, hamcrest produces much better messages about failures; specifically it's better at showing differences between expected and actual data. It can be a bit wordier and does have its own language so there are some downsides too. Also I don't think there's a good equivalent of `assertThrows()` built in.

So the main differences are matters of personal preference.